cost-analyzer/values.yaml

```diff
@@ -2550,6 +2550,10 @@ grafana:
   rbac:
     create: true
 
+  serviceAccount:
+    create: true
+    name:
```

Comment:
Maybe make this `name: ""` to match the convention in other parts of this file.

Reply:
Agree